### ISSUE TITLE
Eizo EV2457

### ISSUE THREAD

**Issue by lameventanas** [openeizo/openeizo#6]
Hi there,

I have an Eizo EV2457, I'm not sure if collecting info from this monitor is still needed.

Also, I would like to know what is the scope of the project.
I'm interested in automatically rotating the display (xrandr or whatever KDE uses) whenever I physically rotate my monitor, it would save me some time.
Is that within the scope?

Cheers!

**Comment by markbolhuis** [comment]
> I have an Eizo EV2457, I'm not sure if collecting info from this monitor is still needed.

Yes please, I'm looking for as much info from the whole range. Can you provide me with the HID descriptor of both the child and parent device. The driver should work for your model, although of course only you can test that.

> Also, I would like to know what is the scope of the project.

The scope is mostly just to expose a child hid device that provides an easier way to send requests. I'd eventually like to have a userspace library. There is no GUI planned.

> I'm interested in automatically rotating the display (xrandr or whatever KDE uses) whenever I physically rotate my monitor, it would save me some time.
> Is that within the scope?

This is not possible, afaik the monitors do not have a sensor to detect orientation. 
You may have to buy a third party usb accelerometer and stick it to the back of the monitor.

**Comment by lameventanas** [comment]
>> I'm interested in automatically rotating the display (xrandr or whatever KDE uses) whenever I physically rotate my monitor, it would save me some time.
>> Is that within the scope?

> This is not possible, afaik the monitors do not have a sensor to detect orientation.
You may have to buy a third party usb accelerometer and stick it to the back of the monitor.

I thought that was how it worked in Windows, after installing the software.

Anyway, what do you need exactly? lsusb -v, or something else?

**Comment by markbolhuis** [comment]
> Anyway, what do you need exactly? lsusb -v, or something else?

Install the driver and hex-dump the contents of the report_descriptor:
```
cd /sys/bus/hid/devices/<dev>
hexdump -e '16/1 "%02x " "\n"' report_descriptor
```
You can then pass that through a hid descriptor decoder.

Make sure to do it to both parent and child device.
